packages/notebook/src/widget.ts

```diff
@@ -841,6 +856,10 @@ export class StaticNotebook extends Widget {
     this._renderedCellsCount++;
   }
 
+  public get cellsToRender(): Map<string, { index: number; cell: Cell }> {
+    return this._toRenderMap;
+  }
```

Comment:
I would rather return the remaining number of cells rather than exposing that sensitive map publicly.

Reply:
I was hesitant on that one while implementing. That should be considered as sensitive, but on the other side the model is anyway exposed, which is even more sensitive. I was thinking that extension developer could benefit from more details to show visuals on unrendered cells. But yes, exposing a simple number sounds safer and simpler, I have implemented that now. Thx.